lib/knowngithub.rb

```diff
@@ -1,5 +1,52 @@
+require 'ipaddr'
+require 'json'
+require 'net/http'
+require 'net/ssh'
+require 'nokogiri'
+require 'open-uri'
 require "knowngithub/version"
 
 module Knowngithub
-  # Your code goes here...
+  def self.fingerprints
+    output = Nokogiri::HTML(open("https://help.github.com/articles/github-s-ssh-key-fingerprints/"))
```

Comment:
you might want to specify something like:
`http.use_ssl = true` to prevent redirect/downgrade to cleartext